doc/source/user_guide/ug_running.rst

```diff
@@ -535,10 +535,10 @@ On macOS:
 
 .. code-block:: bash
 
-  # Download the Miniconda installer to ~/Downloads/miniconda.sh
-  curl -L https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -o ~/Downloads/miniconda.sh
+  # Download the Miniconda installer to ~/miniconda.sh
+  curl -L https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -o ~/miniconda.sh
   # Install Miniconda
-  bash ~/Downloads/miniconda.sh
+  bash ~/miniconda.sh
```

Comment:
Nice to take the time to fix this. If you update your commit message to "Closes https://github.com/CICE-Consortium/CICE/issues/547" instead of "See https://github.com/CICE-Consortium/CICE/issues/547", the issue will be closed automatically when your PR is merged :) 

That could be done for the other issues you mention in previous commits, also, if you wish to close these issues.

Reply:
Just to clarify, I added the "closes" to the PR documentation.  I think this is a good place for it and how I usually do it.  Interesting to know that similar syntax in the commit message could do the same thing.